sources/packages/backend/apps/api/src/esdc-integration/disbursement-receipt-integration/disbursement-receipt-files/disbursement-receipt-file-record.ts

```diff
@@ -1,5 +1,5 @@
 import { getDateOnlyFromFormat } from "@sims/utilities";
-import { DATE_FORMAT } from "../../models/esdc-integration.model";
+import { DATE_FORMAT } from "@sims/integration";
```

Comment:
These date formats are supposed to be integration specific like ESDC, CRA, and Institution. They can evolve differently and must be kept separately.